Confus/Confus/Game.cpp

```diff
@@ -42,6 +42,15 @@ namespace Confus
 			irr::core::vector2df(0.56f, 0.0f), true);
     }
 
+    Game::~Game()
+    {
+        for(size_t i = 0u; i < m_PlayerArray.size(); i++)
+        {
+            //m_PlayerArray[i]->remove();
+            //delete(m_PlayerArray[i]);
+        }
```

Comment:
Let's talk a little bit about this in general on Monday. I guess you can keep it as it is